cfme/tests/infrastructure/test_host.py

```diff
@@ -540,26 +541,51 @@ def test_add_ipmi_refresh(appliance, setup_provider):
 
 
 @test_requirements.infra_hosts
-@pytest.mark.parametrize("crud_action", ['edit_from_hosts', 'edit_from_details', 'remove'])
+@pytest.mark.meta(automates=[1634794])
+@pytest.mark.parametrize("crud_action", ['edit_from_hosts', 'edit_from_details', 'cancel',
+                                         'nav_away_changes', 'nav_away_no_changes', 'remove'])
 def test_infrastructure_hosts_crud(appliance, setup_provider, crud_action):
     """
```

Comment:
Add a description of what the test is doing for the different values of `crud_action`.

Reply:
Added. Let me know if it's clear.